docs/configuration/configuration.md

```diff
@@ -425,11 +432,29 @@ The following meta labels are available on targets during [relabeling](#relabel_
 * `__meta_openstack_private_ip`: the private IP of the OpenStack instance
 * `__meta_openstack_tag_<tagkey>`: each tag value of the instance
 
+#### `instance`
+
+The `instance` role discovers one target per Nova instance. The target
+address defaults to the first private IP address of the instance.
+
+The following meta labels are available on targets during [relabeling](#relabel_config):
+
+* `__meta_openstack_instance_id`: the OpenStack instance ID.
```

Comment:
Nitpick: The bullet points above don't include periods at the end. Can we remove them here for consistency?

Reply:
I realize there isn't consistency in regards to this throughout the docs for this but it would be nice to have a bit of consistency for each section of this page at least :)